components/fireflies/actions/find-meeting-by-id/find-meeting-by-id.mjs

```diff
@@ -26,6 +31,11 @@ export default {
         },
       },
     });
+
+    if (meeting.errors?.length) {
```

Comment:
So, minor coding style nit here, but rather than requiring all callers of [`proxy`](https://github.com/PipedreamHQ/pipedream/blob/ea0ddf2c66cacd0e3cec38dc47dfd8434862ade3/components/fireflies/fireflies.app.mjs#L73) handle these errors, which can be error prone, it might be more consistent to have the base function detect an error response (e.g. by finding an `error` key in the json response), and raising the `ConfigurationError` at that point. That way `upload-audio` any future actions, as well as async options can all benefit from the same error handling. Does that make sense?